en/docs/reference/pricing.md

```diff
@@ -2,7 +2,7 @@
 
 ## Introduction
 
-Choreo is a Digital Platform as a Service that helps you develop integration solutions faster by reducing the coding effort with a low-code development platform. 
+Choreo is a Digital Platform as a Service that helps you develop cloud-native applications by focusing on innovation and increasing developer productivity. 
```

Comment:
We don't need to have "digital platform as a service" in title caps right?

Also, I recall deciding to go without the hyphen for the term "cloud native".

Reply:
fixed